brainmap-cms/settings_custom.py

```diff
@@ -56,7 +67,10 @@
     "True"
 ]
 
-BRANDING = [ _SGCI_BRANDING, _TACC_BRANDING, _UTEXAS_BRANDING ]
+# BRANDING = [ _UTHSCSA_BRANDING, _SGCI_BRANDING, _TACC_BRANDING, _UTEXAS_BRANDING ]
+# BRANDING = [ _SGCI_BRANDING, _UTHSCSA_BRANDING, _TACC_BRANDING, _UTEXAS_BRANDING ]
+# BRANDING = [ _SGCI_BRANDING, _TACC_BRANDING, _UTHSCSA_BRANDING, _UTEXAS_BRANDING ]
+BRANDING = [ _SGCI_BRANDING, _TACC_BRANDING, _UTEXAS_BRANDING, _UTHSCSA_BRANDING ]
```

Comment:
__Pending Design Decision__
See https://github.com/TACC/Core-CMS/pull/414#user-content-fnref-2-b6a6fd48ee3671fa2238e2a7946fe9ba (might need to scroll up a bit on that page).

Reply:
Going with position 4.